clash-lib/tests/Clash/Tests/Scaffold.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE PolyKinds             #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeApplications      #-}
+{-# LANGUAGE MagicHash             #-}
+{-# LANGUAGE TypeInType             #-}
+
+module Clash.Tests.Scaffold where
+
+import           Test.Tasty
+import           Test.Tasty.HUnit
+
+import           Clash.Primitives.Scaffold
+import           Clash.Prelude hiding (undefined)
+
+makeScaffold "testFunction" "testPrimitive" []
+  [ [ ClkOut "d1clk1"
+    , ClkIn  "d1clk2"
+    , In     "d1i1" 1
+    , Out    "d1o1" 2
+    ]
+  , [ ClkOut "d2clk1"
+    , ClkIn  "d2clk2"
+    , In     "d2i1" 1
+    , In     "d2i2" 1
+    , Out    "d2o1" 2
+  ] ]
+
+-- | Type check our generated functions/datatypes
+exists :: Bool
+exists = True
```

Comment:
Should this simply be `True`? If so can you leave a comment stating this and the rationale?